[2023-11-06T12:06:04.086+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-22T00:00:00+00:00 [queued]>
[2023-11-06T12:06:04.120+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-22T00:00:00+00:00 [queued]>
[2023-11-06T12:06:04.122+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:06:04.258+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-10-22 00:00:00+00:00
[2023-11-06T12:06:04.265+0000] {standard_task_runner.py:57} INFO - Started process 861 to run task
[2023-11-06T12:06:04.274+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-10-22T00:00:00+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmp05w87jam']
[2023-11-06T12:06:04.280+0000] {standard_task_runner.py:85} INFO - Job 96: Subtask task_a
[2023-11-06T12:06:04.404+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-22T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:06:04.568+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-10-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-22T00:00:00+00:00'
[2023-11-06T12:06:04.570+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:06:04.572+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World :)']
[2023-11-06T12:06:04.591+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:06:04.599+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: syntax error near unexpected token `)'
[2023-11-06T12:06:04.600+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: `echo Hello World :)'
[2023-11-06T12:06:04.601+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-11-06T12:06:04.623+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-11-06T12:06:04.634+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231022T000000, start_date=20231106T120604, end_date=20231106T120604
[2023-11-06T12:06:04.772+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 96 for task task_a (Bash command failed. The command returned a non-zero exit code 2.; 861)
[2023-11-06T12:06:04.814+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-06T12:06:04.859+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-06T12:19:07.261+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-22T00:00:00+00:00 [queued]>
[2023-11-06T12:19:07.282+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-22T00:00:00+00:00 [queued]>
[2023-11-06T12:19:07.286+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:19:07.324+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-10-22 00:00:00+00:00
[2023-11-06T12:19:07.331+0000] {standard_task_runner.py:57} INFO - Started process 2063 to run task
[2023-11-06T12:19:07.339+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-10-22T00:00:00+00:00', '--job-id', '168', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpoyv1nmdd']
[2023-11-06T12:19:07.349+0000] {standard_task_runner.py:85} INFO - Job 168: Subtask task_a
[2023-11-06T12:19:07.442+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-22T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:19:07.581+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-10-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-22T00:00:00+00:00'
[2023-11-06T12:19:07.582+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:19:07.584+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World.']
[2023-11-06T12:19:07.595+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:19:07.598+0000] {subprocess.py:93} INFO - Hello World.
[2023-11-06T12:19:07.599+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-06T12:19:07.636+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231022T000000, start_date=20231106T121907, end_date=20231106T121907
[2023-11-06T12:19:07.676+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-06T12:19:07.701+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
